static/src/javascripts/projects/common/modules/onward/geo-most-popular.js

```diff
@@ -33,12 +34,30 @@ class GeoMostPopular extends Component {
     }
 }
 
-const geoMostPopular = {
-    render: once((): Promise<void> => {
+// we don't want to show most popular on short articles as the sticky right mpu slot will push most popular behind other containers at the bottom.
+const fetchMostPopular = (articleBodyHeight: number): void => {
+    if (articleBodyHeight > 1500) {
```

Comment:
might want to label this value with a `const`

Reply:
Done.